services/src/main/java/org/keycloak/services/resources/admin/RealmAdminResource.java

```diff
@@ -743,7 +743,7 @@ public void deleteSession(@PathParam("session") String sessionId, @DefaultValue(
     @Operation(summary = "Get client session stats Returns a JSON map.",
         description = "The key is the client id, the value is the number of sessions that currently are active with that client. Only clients that actually have a session associated with them will be in this map.")
     @APIResponses(value = {
-        @APIResponse(responseCode = "200", description = "", content = @Content(schema = @Schema(implementation = Map.class, type = SchemaType.ARRAY))),
+        @APIResponse(responseCode = "200", description = "OK"),
```

Comment:
@ahus1 here you have removed the content, why? In the rest of the endpoints it is always set